src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -28,8 +29,9 @@
     public FindCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_ITEM);
 
-        if (!argMultimap.getValue(PREFIX_ITEM).isPresent()) {
-            throw new ParseException(Item.MESSAGE_CONSTRAINTS);
+        if (argMultimap.getPreamble().isEmpty() && !argMultimap.getValue(PREFIX_ITEM).isPresent()) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                    FindCommand.MESSAGE_USAGE));
```

Comment:
So do we now require users to specify the preamble?

What is the command format for `find` now?